inc/options/install.php

```diff
@@ -1,100 +1,422 @@
+					'acronym' => array(
+						'title' => array(),
+					),
+				)
+			);
+			?>
+		</li>
+		<li>
+			<?php
+			echo wp_kses(
+				sprintf(
+					// translators: 1 opening HTML em tag, 2 closing HTML em tag,
+					// translators: 3 opening HTML a tag to W3TC DatabaseCache admin page, 4 closing HTML a tag.
+					__(
+						'%1$sOptional:%2$s On the "%3$sDatabase Cache%4$s" tab the recommended settings are preset. If using a shared hosting account use the "disk" method with caution; in either of these cases the response time of the disk may not be fast enough, so this option is disabled by default.',
+						'w3-total-cache'
+					),
+					'<em>',
+					'</em>',
+					'<a href="admin.php?page=w3tc_dbcache">',
+					'</a>'
+				),
+				array(
+					'em' => array(),
+					'a'  => array(
+						'href' => array(),
+					),
+				)
+			);
+			?>
+		</li>
+		<li>
+			<?php
+			echo wp_kses(
+				sprintf(
+					// translators: 1 opening HTML em tag, 2 closing HTML em tag,
+					// translators: 3 opening HTML a tag to W3TC ObjectCache admin page, 4 closing HTML a tag.
+					__(
+						'%1$sOptional:%2$s On the "%3$sObject Cache%4$s" tab the recommended settings are preset. If using a shared hosting account use the "disk" method with caution, the response time of the disk may not be fast enough, so this option is disabled by default. Test this option with and without database cache to ensure that it provides a performance increase.',
+						'w3-total-cache'
+					),
+					'<em>',
+					'</em>',
+					'<a href="admin.php?page=w3tc_objectcache">',
+					'</a>'
+				),
+				array(
+					'em' => array(),
+					'a'  => array(
+						'href' => array(),
+					),
+				)
+			);
+			?>
+		</li>
+		<li>
+			<?php
+			echo wp_kses(
+				sprintf(
+					// translators: 1 opening HTML em tag, 2 closing HTML em tag,
+					// translators: 3 opening HTML a tag to W3TC Cache Groups admin page, 4 closing HTML a tag.
+					__(
+						'%1$sOptional:%2$s On the "%3$sUser Agent Groups%4$s" tab, specify any user agents, like mobile phones if a mobile theme is used.',
+						'w3-total-cache'
+					),
+					'<em>',
+					'</em>',
+					'<a href="admin.php?page=w3tc_cachegroups">',
+					'</a>'
+				),
+				array(
+					'em' => array(),
+					'a'  => array(
+						'href' => array(),
+					),
+				)
+			);
+			?>
+		</li>
+	</ol>
 
-    <p>
-    	<?php _e( 'Check out the <acronym title="Frequently Asked Questions">FAQ</acronym> for more details on <a href="admin.php?page=w3tc_faq">usage</a>', 'w3-total-cache' ); ?>.
-    </p>
+	<p>
+		<?php
+		echo wp_kses(
+			sprintf(
+				// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag,
+				// translators: 3 opening HTML acronym tag, 4 closing HTML acronym tag.
+				__(
+					'Check out the %1$sFAQ%2$s for more details on %3$susage</a>.',
+					'w3-total-cache'
+				),
+				'<acronym title="' . __( 'Frequently Asked Questions', 'w3-total-cache' ) . '">',
+				'</acronym>',
+				'<a href="admin.php?page=w3tc_faq">',
```

Comment:
Use `esc_url()` and `admin_url()` for the `href`.

Reply:
Added esc_url and admin_url to href